test/postgresql.test.js

```diff
@@ -206,6 +206,15 @@ describe('postgresql connector', function () {
           console.warn.restore();
         });
 
+        it('should work', function(done) {
+          Post.find({where: {content: {regexp: '^a/i'}}}, function(err, posts) {
```

Comment:
Also its not really just the leading slash, its more "you don't need to provide regex delimiters for regex strings" (meaning the leading and trailing forward slashes).